test/integration/controllers/etcd/reconciler_test.go

```diff
@@ -605,13 +605,13 @@ func validateDefaultValuesForEtcd(instance *druidv1alpha1.Etcd, s *appsv1.Statef
 			"Name":      Equal(instance.Name),
 			"Namespace": Equal(instance.Namespace),
 			"Annotations": MatchAllKeys(Keys{
-				"gardener.cloud/owned-by":   Equal(fmt.Sprintf("%s/%s", instance.Namespace, instance.Name)),
-				"gardener.cloud/owner-type": Equal("etcd"),
-				"app":                       Equal("etcd-statefulset"),
-				"role":                      Equal("test"),
-				"instance":                  Equal(instance.Name),
-				"checksum/etcd-configmap":   Equal(configMapChecksum),
-				"name":                      Equal("etcd"),
+				common.GardenerOwnedBy:    Equal(fmt.Sprintf("%s/%s", instance.Namespace, instance.Name)),
```

Comment:
These changes are completely unrelated to etcdcopybackuptask and should not be clubbed with this PR. Please remove this and file a separate PR for this.